stackable-base/Dockerfile

```diff
@@ -113,3 +110,30 @@ RUN update-ca-trust && \
 COPY --from=product-utils-builder --chown=stackable:stackable /config-utils/target/release/config-utils /stackable/config-utils
 COPY --from=product-utils-builder --chown=stackable:stackable /config-utils/config-utils.cdx.xml /stackable/config-utils.cdx.xml
 ENV PATH="${PATH}:/stackable"
+
+# These labels have mostly been superceded by the OpenContainer spec annotations below but it doesn't hurt to include them
+LABEL maintainer="info@stackable.tech"
+LABEL vendor="Stackable GmbH"
+
+# Overwriting/Pinning UBI labels
+# https://github.com/projectatomic/ContainerApplicationGenericLabels
+LABEL vcs-ref=""
+LABEL distribution-scope="public"
+LABEL url="https://stackable.tech"
+ARG TARGETARCH
+LABEL architecture="${TARGETARCH}"
+LABEL com.redhat.component=""
+# It complains about it being an invalid label but RedHat uses it and we want to override it and it works....
+# hadolint ignore=DL3048
+LABEL com.redhat.license_terms=""
+LABEL io.buildah.version=""
+LABEL io.openshift.expose-services=""
+
+# https://github.com/opencontainers/image-spec/blob/036563a4a268d7c08b51a08f05a02a0fe74c7268/annotations.md#annotations
+LABEL org.opencontainers.image.authors="info@stackable.tech"
+LABEL org.opencontainers.image.url="https://stackable.tech"
+LABEL org.opencontainers.image.source="https://github.com/stackabletech/docker-images/"
+LABEL org.opencontainers.image.vendor="Stackable GmbH"
+LABEL org.opencontainers.image.licenses="Apache-2.0"
+
+COPY --chown=stackable:stackable shared/npm/.npmrc /stackable/
```

Comment:
I don't get this line. Why is it needed and where is it `.npmrc` copied from?

Reply:
Oh, damn. That is a mistake and shoud not be in there. This is a test to try the build-repo. I'll remove it. Good catch. Thank you!

Reply:
[`4c7dc6e` (#731)](https://github.com/stackabletech/docker-images/pull/731/commits/4c7dc6e9287d15dbf2d45494d718616ba5920843)